### PR TITLE
VACMS-13736 Update PW portfolio README while learning about PW products

### PR DESCRIPTION
**Pull request by randimays**
Updating Public Websites portfolio documentation. Fixing typos, dead links, etc.

### CODE REVIEW

File: products/public-websites/README.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ Campaign Landing Pages are intended to prevent creation of standalone marketing
 **More info:**
 * A6 program GDrive folder:[ Campaign Landing Page](https://drive.google.com/drive/u/1/folders/1pyvDhf6ZdvBehFs3lFzyAF2kMeGkazX6)
 * [Product launch docs: tier 2 content IA & Design > Campaign Landing Page Templates - Github](https://github.com/department-of-veterans-affairs/va.gov-team/tree/69833737d9fe22b8990bb987e7c50de13205c5d5/products/content/tier-2-content-IA-and-design/campaign-landing-page-templates) 
-* [Public Websites team documentation: Campaign Landing Pages](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/public-websites/campaign-landing-pages/campaign-landing-pages.md) - 
+* [Public Websites team documentation: Campaign Landing Pages](https://github.com/department-of-veterans-affairs/va.gov-team/blob/fb49a95fea9d755534cb7f6a77060e221625d5a1/products/campaign-landing-page/campaign-landing-page-templates/state-of-product.md)
```
Comment (randimays) [inline products/public-websites/README.md:115]: Was a dead link. Is this the correct new link?

File: products/public-websites/README.md [hunk 2]
```diff
@@ -511,10 +512,10 @@ VA.gov header and footer, within VA.gov context, including presentation of banne
 * URL: [https://www.va.gov/](https://www.va.gov/) 
 
 **More info:**
-* 2019 research for launch: [https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/global/header-footer-and-navigation](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/global/header-footer-and-navigation)
+* 2019 research for launch: [https://github.com/department-of-veterans-affairs/va.gov-team/blob/fb49a95fea9d755534cb7f6a77060e221625d5a1/products/information-architecture/archive/2019-2020-navigation-docs/primary-header-footer/2019-header-nav-outline.md](https://github.com/department-of-veterans-affairs/va.gov-team/blob/fb49a95fea9d755534cb7f6a77060e221625d5a1/products/information-architecture/archive/2019-2020-navigation-docs/primary-header-footer/2019-header-nav-outline.md)
```
Comment (randimays) [inline products/public-websites/README.md:515]: Was a dead link. Is this the correct new link?

File: products/public-websites/README.md [hunk 3]
```diff
@@ -511,10 +512,10 @@ VA.gov header and footer, within VA.gov context, including presentation of banne
...
 * Mobile authed header: [https://github.com/department-of-veterans-affairs/va.gov-team/issues/30626](https://github.com/department-of-veterans-affairs/va.gov-team/issues/30626)
 * Mobile unauthed header: [https://github.com/department-of-veterans-affairs/va.gov-team/issues/30623](https://github.com/department-of-veterans-affairs/va.gov-team/issues/30623) 
-* [VFS Product Directory: Site Navigation](https://depo-platform-documentation.scrollhelp.site/getting-started/site-navigation)
+* [VFS Product Directory: Site Navigation](https://depo-platform-documentation.scrollhelp.site/getting-started/site-navigation) TODO dead link
```
Comment (randimays) [inline products/public-websites/README.md:518]: Dead link. What is the new link?

File: products/public-websites/README.md [hunk 4]
```diff
@@ -572,7 +573,7 @@ VA.gov uses search.gov as its engine. Public Websites owns that integration, and
 
 **Example content:**
 * URL: [https://www.va.gov/search/?query=test&t=false](https://www.va.gov/search/?query=test&t=false)  
-* Example issue: 
+* Example issue: TODO
```
Comment (randimays) [inline products/public-websites/README.md:576]: What should this issue be?

File: products/public-websites/README.md [hunk 5]
```diff
@@ -582,7 +583,7 @@ VA.gov uses search.gov as its engine. Public Websites owns that integration, and
 * [On Site Search - product docs](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/on-site-search)
 * Design system: [Search input](https://design.va.gov/components/search-input)
 * Search.gov indexing documentation: https://search.gov/indexing/indexing-with-searchgov.html 
-* 2020: bring more sites under search banner: [https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/global/search](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/global/search) 
+* 2020: bring more sites under search banner: [https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/global/search](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/global/search) TODO dead link
```
Comment (randimays) [inline products/public-websites/README.md:586]: The `global` folder no longer exists. I poked around for a while trying to find docs that could fit here. What is the new link for this?

File: products/public-websites/README.md [hunk 6]
```diff
@@ -596,9 +597,9 @@ Request to change a URL and/or implement a redirect for a URL. Also for vanity U
 Facilities team can handle redirects for Facility URLs / content. 
 
 **Workflow:**
-[Redirects IA Context & Workflow](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/request-redirect.md)
+[Redirects IA Context & Workflow](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/request-redirect.md) TODO dead link
 * Another team requests a URL redirect
-* Sitewide Content team vets the request – IA will approve the requested URLs (Process: [Platform IA Redirects](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/request-redirect.md))
+* Sitewide Content team vets the request – IA will approve the requested URLs (Process: [Platform IA Redirects](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/request-redirect.md)) TODO dead link
```
Comment (randimays) [inline products/public-websites/README.md:602]: I poked around for a while trying to find docs that could fit here. What is the new link for this?

File: products/public-websites/README.md [hunk 7]
```diff
@@ -618,7 +619,7 @@ Facilities team can handle redirects for Facility URLs / content.
 **More info:**
 * [Platform Redirect Implementation strategy](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/redirect-implementation-strategy.md) - context, describes various types of redirects, and mechanisms for carrying them out, including server-side vs. client-side.
   * [Redirects(sensitive repo)](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/vsa/teams/public-websites/redirects.md) - this doc will be merged into the above Implementation Strategy doc 
-* [Redirects IA Context & Workflow](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/request-redirect.md)
+* [Redirects IA Context & Workflow](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/request-redirect.md) TODO dead link
```
Comment (randimays) [inline products/public-websites/README.md:622]: Same link as above (for redirects)

File: products/public-websites/README.md [hunk 8]
```diff
@@ -647,7 +648,7 @@ Calls to action on these widgets point users to the Electronic Healthcare Record
 * [React widget flows](https://app.mural.co/t/vagov6717/m/vagov6717/1651166973924/bc8bc44cab4e4ae7b84819d22ba89b11aa70e5e1?invited=true) (Mural)
 * **Q2/Q3 2022 project:** Active effort to replace old hard coded data file with Drupal as source of truth for which healthcare records management system is used by a VA medical center system. 
    * [CMS source of truth documentation](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/public-websites/Cerner-Support/cms-source-of-truth) (github)
-* Cerner cutovers have been complex and not additional facilities will move to Cerner in 2022. Cerner cutovers should resume in 2023.
+* Cerner cutovers have been complex and now additional facilities will move to Cerner in 2022. Cerner cutovers should resume in 2023.
```
Comment (randimays) [inline products/public-websites/README.md:651]: Is this the correct word?

File: products/public-websites/README.md [hunk 9]
```diff
@@ -667,7 +668,8 @@ The homepage includes links to key traffic areas of the site. These links are re
 * A6 program GDrive folder:[ Homepage](https://drive.google.com/drive/u/1/folders/1a-dg6-nWuqNTxO9lds4iqWHo5DxYj_z8)
 * **Q2/Q3 2022 project:**[Homepage redesign iteration:](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/public-websites/home-page/home-page-redesign-initiative/README.md#home-page-redesign-iteration) design refresh underway, prototype in staging.  Q3 user testing & will lead to design / build iteration for further testing. 
    * [[Epic] VA.gov Home page redesign #40845](https://github.com/department-of-veterans-affairs/va.gov-team/issues/40845) 
-* 2021 Top Tasks research/info: [https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/global/home-page](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/global/home-page)
```
Comment (randimays) [inline products/public-websites/README.md:670]: Dead link. `global` folder is gone. Couldn't find anything for 2021 Top Tasks, but I found the two below 2022 tasks. Is this ok?

File: products/public-websites/README.md [hunk 10]
```diff
@@ -733,7 +735,6 @@ https://www.va.gov/discharge-upgrade-instructions/questions
 The discharge update wizard was written by the Decision Tools team. That team transitioned responsibility to the Search & Discovery Team, which has since become Public Websites purview. 
 
 **More info:**
-* vets.gov-team repo: [discharge tool issues](https://github.com/department-of-veterans-affairs/vets.gov-team/issues?q=label%3A%22discharge+tool%22+)
```
Comment (randimays) [inline products/public-websites/README.md:736]: Changed to `va.gov-team` and filtered issues by `Discharge Upgrade Wizard` label (`discharge tool` is not a label there anymore). No issues found.

File: products/public-websites/README.md [hunk 11]
```diff
@@ -777,7 +778,7 @@ Still owned by PW:
 * https://www.va.gov/disability/how-to-file-claim
 * https://www.va.gov/decision-reviews/higher-level-review/
 
-TODO: Learn what is the actual widget component of these pages, and PW ownership of that. Have these all been replaced by online forms? 
+TODO: Learn what is the actual widget component of these pages, and PW ownership of that. Have these all been replaced by online forms? TODO?
```
Comment (randimays) [inline products/public-websites/README.md:781]: Little bit more context on these URLs. How do we want to fill in this question?

`https://www.va.gov/education/eligibility`
- React component: src/applications/edu-benefits/component/createEducationApplicationStatus.jsx

`https://www.va.gov/education/how-to-apply`
- React component: src/applications/edu-benefits/component/createEducationApplicationStatus.jsx

`https://www.va.gov/disability/eligibility`
- No React component on this page

`https://www.va.gov/disability/how-to-file-claim`
- No React component on this page

`https://www.va.gov/decision-reviews/higher-level-review`
- No React component on this page

File: products/public-websites/README.md (hunk 1 above)
Comment (jilladams) [reply products/public-websites/README.md:115]: We can just remove this line - good catch, and I don't think we need to point anywhere that isn't already linked in other places in this section.

File: products/public-websites/README.md [hunk 12]
```diff
@@ -233,26 +233,27 @@ Announcements are reusable components that give a common way to make full page a
...
+  },
+],
+```
 
-New announcement components should be built in the announcements components directory (src/platform/site-wide/announcements/components), although the announcement framework allows any React component in vets-website to be used.
+New announcement components should be built in the announcements components directory (`src/platform/site-wide/announcements/components`), although the announcement framework allows any React component in vets-website to be used.
 
 The `announcement` property in the rendered component will contain the announcement as stored in the config, so in this case `announcement.name` will render `New Education Feature`.
 
-The `show` perperty can be set to one of 3 values defined in the AnnouncementBehavior enum constant, which are SHOW_ONCE, SHOW_EVERY_TIME and SHOW_ONCE_PER_SESSION. The SHOW_ONCE option persists to `localStorage`, the SHOW_ONCE_PER_SESSION persists to the `sessionStorage` nad the SHOW_EVERY_TIME does not persist dismissal at all.
```
Comment (jilladams) [inline products/public-websites/README.md:255]: `perperty` is my fave typo so far this week

File: products/public-websites/README.md (hunk 2 above)
Comment (jilladams) [reply products/public-websites/README.md:515]: Let's remove this. Dave has overhauled the existing docs, and I dont' think we need this pointer anymore.

File: products/public-websites/README.md (hunk 3 above)
Comment (jilladams) [reply products/public-websites/README.md:518]: Plat seems to have removed nav from https://depo-platform-documentation.scrollhelp.site/getting-started/vfs-product-directory, so I think we just remove this.

File: products/public-websites/README.md [hunk 13]
```diff
@@ -546,7 +547,7 @@ Veterans are eligible for benefits based on a complex matrix of factors includin
 
 **More info:**
 * [Income Limits App product documentation](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/income-limits-app) in Github
-* A6 program GDrive folder: [Benefits Eligibility Tool](https://drive.google.com/drive/u/1/folders/1FvuprvP7-6tpn8jrctRrms8MTOqPvuAL)
+* A6 program GDrive folder: [Benefits Eligibility Tool](https://drive.google.com/drive/u/1/folders/1FvuprvP7-6tpn8jrctRrms8MTOqPvuAL) TODO dead link
```
[the remark this answers is not in the document]
Comment (jilladams) [reply products/public-websites/README.md:550]: This got a real name, so I **believe** this should go to https://drive.google.com/drive/u/1/folders/1uwHUEe8UBHST-didMR68Xfaud-BaI0iJ

File: products/public-websites/README.md [hunk 14]
```diff
@@ -558,10 +559,10 @@ Veterans are eligible for benefits based on a complex matrix of factors includin
 The VA is comprised of multiple administrative offices, e.g. VHA, VEO, NCA, VBA, etc. ([diagram](https://app.mural.co/t/vagov6717/m/vagov6717/1643724063628/1654203c1a0d6c763bbd5c969febf94ec476df79?sender=kevinwalsh8610)) Each of these offices contributes their own content to the Veteran-facing pages of the modernized VA.gov CMS. However, information _about_ the offices themselves is not necessarily a Veteran priority, e.g. organization of, administrative officials, etc. The Public Websites team intends to build a product that organizes information about these Offices, synced from Salesforce, and displays that content within VA.gov somewhere. 
 
 **Example content:**
-* TBD
+* TBD TODO?
```
[the remark this answers is not in the document]
Comment (jilladams) [reply products/public-websites/README.md:562]: Let's remove the entire Offices section. We plan to deprecate: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14404

File: products/public-websites/README.md (hunk 4 above)
Comment (jilladams) [reply products/public-websites/README.md:576]: Still not sure on this. I think it's ok to leave the TODO, or we can remove the whole line.

File: products/public-websites/README.md (hunk 5 above)
Comment (jilladams) [reply products/public-websites/README.md:586]: Let's remove this line. It's a dupe of 3 bullets up.

File: products/public-websites/README.md [hunk 15]
```diff
@@ -596,9 +597,9 @@ Request to change a URL and/or implement a redirect for a URL. Also for vanity U
 Facilities team can handle redirects for Facility URLs / content. 
 
 **Workflow:**
-[Redirects IA Context & Workflow](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/request-redirect.md)
+[Redirects IA Context & Workflow](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/request-redirect.md) TODO dead link
```
[the remark this answers is not in the document]
Comment (jilladams) [reply products/public-websites/README.md:600]: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/process/redirects.md.  (I cloned the team repo locally to find this)

File: products/public-websites/README.md (hunk 6 above)
Comment (jilladams) [reply products/public-websites/README.md:602]: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/process/redirects.md

File: products/public-websites/README.md (hunk 7 above)
Comment (jilladams) [reply products/public-websites/README.md:622]: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/process/redirects.md

File: products/public-websites/README.md (hunk 8 above)
Comment (jilladams) [reply products/public-websites/README.md:651]: Should be `no` -- effort is on pause

File: products/public-websites/README.md (hunk 11 above)
Comment (jilladams) [reply products/public-websites/README.md:781]: Should we just add that info into the page? Seems useful.

**Comment by jilladams** [comment]
@randimays review is done! Thank you thank you thank you -- what an amazing side benefit of your downtime research. Also what an amazing demonstration of why Github needs to figure out relative links / redirects for when .md files move paths. UGH.

File: products/public-websites/README.md (hunk 1 above)
Comment (randimays) [reply products/public-websites/README.md:115]: 👍

File: products/public-websites/README.md (hunk 2 above)
Comment (randimays) [reply products/public-websites/README.md:515]: 👍

File: products/public-websites/README.md (hunk 3 above)
Comment (randimays) [reply products/public-websites/README.md:518]: 👍

File: products/public-websites/README.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (randimays) [reply products/public-websites/README.md:550]: 👍

File: products/public-websites/README.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (randimays) [reply products/public-websites/README.md:562]: 👍

File: products/public-websites/README.md (hunk 4 above)
Comment (randimays) [reply products/public-websites/README.md:576]: 👍

File: products/public-websites/README.md (hunk 5 above)
Comment (randimays) [reply products/public-websites/README.md:586]: 👍

File: products/public-websites/README.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (randimays) [reply products/public-websites/README.md:600]: 👍

File: products/public-websites/README.md (hunk 6 above)
Comment (randimays) [reply products/public-websites/README.md:602]: 👍

File: products/public-websites/README.md (hunk 7 above)
Comment (randimays) [reply products/public-websites/README.md:622]: 👍

File: products/public-websites/README.md (hunk 8 above)
Comment (randimays) [reply products/public-websites/README.md:651]: 👍

**Review by jilladams — APPROVED** [review]

**Pull request merged**